docs/pages/router/migrate/from-react-navigation.mdx

```diff
@@ -0,0 +1,451 @@
+
+When you use groups or shared routes, you can navigate to specific tabs by using the fully qualified route name, e.g. `/(home)/settings` instead of `/settings`.
+
+### Migrating screen tracking events
+
+You may have your screen tracking setup according to our [React Navigation screen tracking guide](https://reactnavigation.org/docs/screen-tracking/), update it according to the [Expo Router screen tracking guide](/router/reference/screen-tracking).
+
+### Using platform-specifc components for screens
+
+Refer to the [platform-specific modules](/router/advance/platform-specific-modules) guide for info on switching UI based on the platform.
+
+### Replacing the NavigationContainer
+
+The global React Navigation [`<NavigationContainer />`](https://reactnavigation.org/docs/navigation-container/) is completely managed in Expo Router. Expo Router provides systems for achieving the same functionality as the `NavigationContainer` without needing to use it directly.
+
+<Collapsible summary="API substitutions">
+
+### ref
+
+The `NavigationContainer` ref should not be accessed directly. Use the following methods instead.
+
+#### `resetRoot​`
+
+Navigate to the initial route of the application. For example, if your app starts at `/` (recommended), then you can replace the current route with `/` using this method.
+
+```js
+import { useRouter } from 'expo-router';
+
+function Example() {
+  const router = useRouter();
+
+  return (
+    <Text
+      onPress={() => {
+        // Go to the initial route of the application.
+        router.replace('/');
+      }}>
+      Reset App
+    </Text>
+  );
+}
+```
+
+#### `getRootState`
+
+Use `useRootNavigationState()`.
+
+#### `getCurrentRoute`
+
+Unlike React Navigation, Expo Router can reliably represent any route with a string. Use the [`usePathname()`](/docs/features/linking#usepathname) or [`useSegments()`](/docs/features/linking#usesegments) hooks to identify the current route.
+
+#### `getCurrentOptions`
+
+Use the [`useLocalSearchParams()`](/docs/features/linking#useLocalSearchParams) hook to get the current route's query parameters.
+
+#### `addListener`
+
+The following events can be migrated:
+
+#### `state`
+
+Use the [`usePathname()`](/docs/features/linking#usepathname) or [`useSegments()`](/docs/features/linking#usesegments) hooks to identify the current route. Use in conjunction with `useEffect(() => {}, [...])` to observe changes.
+
+#### `options`
+
+Use the [`useLocalSearchParams()`](/docs/features/linking#useLocalSearchParams) hook to get the current route's query parameters. Use in conjunction with `useEffect(() => {}, [...])` to observe changes.
+
+### props
+
+Migrate the following `<NavigationContainer />` props:
+
+#### `initialState`
+
+In Expo Router, you can rehydrate your application state from a route string (e.g. `/user/evanbacon`). Use [redirects](/docs/features/linking#redirect) to handle initial states. See [shared routes](/docs/features/shared-routes) for advanced redirects.
```

Comment:
```suggestion
In Expo Router, you can rehydrate your application state from a route string (for example, `/user/evanbacon`). Use [redirects](/docs/features/linking#redirect) to handle initial states. See [shared routes](/router/advanced/shared-routes/) for advanced redirects.
```

Couldn't find a link for: /docs/features/linking#redirect after migration so not sure what should go there